MobileFilter/sections/specific_web.txt

```diff
@@ -53,6 +53,7 @@ independent.co.uk,komputerswiat.pl,dobreprogramy.pl,amp.pcwelt.de,tomshardware.c
 avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com##.inline-ad-container
 !
 loveheaven.net##center > a[target="_blank"] > img
+vnexpress.net##a[href="https://apps.apple.com/vn/app/bluezone/id1508062685?ls=1"]
```

Comment:
It is a third-party app and I don't see it on a desktop so I've put the rule in specific_web.txt

Reply:
missed 
![image](https://user-images.githubusercontent.com/8361299/91157604-367c7900-e6ce-11ea-9eb1-6ea90b37146b.png)